src/generic/LaurentMPoly.jl

```diff
@@ -368,6 +368,10 @@ function coefficients(a::LaurentMPolyWrap)
     return coefficients(a.mpoly)
 end
 
+function constant_coefficient(a::LaurentMPolyWrap)
+    return coeff(a.mpoly, -a.mindegs)
```

Comment:
Hmm, if `a.mindegs` has a positive entry, not sure if passing a negative entry to `coeff` is a good idea.